### PR TITLE
new plugin: Flow.Launcher.Plugin.IPDetails

### PR DESCRIPTION
**Pull request by ibnuh**
Please see more details here: https://github.com/ibnuh/Flow.Launcher.Plugin.IPDetails

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -2362,5 +2362,20 @@
...
+        "Name": "IP Details",
+        "Description": "Retrieve current public IP details/lookup or by providing an IP address. Optionally, include an API key for ipapi.is to take advantage of up to 1000 free requests per day.",
+        "Author": "ibnuh",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/ibnuh/Flow.Launcher.Plugin.IPDetails",
+        "UrlDownload": "https://github.com/ibnuh/Flow.Launcher.Plugin.IPDetails/releases/download/1.0.0/Flow.Launcher.Plugin.IPDetails.zip",
+        "UrlSourceCode": "https://github.com/ibnuh/Flow.Launcher.Plugin.IPDetails/tree/main",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/ibnuh/Flow.Launcher.Plugin.IPDetails/Flow.Launcher.Plugin.IPDetails/images/icon.png",
+        "Tested": true,
```
Comment (jjw24) [inline plugins.json:2377]:
```suggestion
```

File: plugins.json [hunk 2]
```diff
@@ -2362,5 +2362,20 @@
...
+        "DateAdded": "2024-06-09T07:13:50Z",
```
Comment (jjw24) [inline plugins.json:2378]:
```suggestion
```

File: plugins.json [hunk 3]
```diff
@@ -2362,5 +2362,20 @@
...
+        "ID": "EEC889D32E2A4F7FA7153C32B0C048F6",
+        "Name": "IP Details",
+        "Description": "Retrieve current public IP details/lookup or by providing an IP address. Optionally, include an API key for ipapi.is to take advantage of up to 1000 free requests per day.",
+        "Author": "ibnuh",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/ibnuh/Flow.Launcher.Plugin.IPDetails",
+        "UrlDownload": "https://github.com/ibnuh/Flow.Launcher.Plugin.IPDetails/releases/download/1.0.0/Flow.Launcher.Plugin.IPDetails.zip",
+        "UrlSourceCode": "https://github.com/ibnuh/Flow.Launcher.Plugin.IPDetails/tree/main",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/ibnuh/Flow.Launcher.Plugin.IPDetails/Flow.Launcher.Plugin.IPDetails/images/icon.png",
```
Comment (jjw24) [inline plugins.json:2376]:
```suggestion
        "IcoPath": "https://cdn.jsdelivr.net/gh/ibnuh/Flow.Launcher.Plugin.IPDetails/Flow.Launcher.Plugin.IPDetails/images/icon.png"
```

File: plugins.json [hunk 4]
```diff
@@ -2362,5 +2362,20 @@
...
+        "Tested": true,
+        "DateAdded": "2024-06-09T07:13:50Z",
+        "LatestReleaseDate": "2024-06-09T07:13:50Z"
```
Comment (jjw24) [inline plugins.json:2379]:
```suggestion
```

**Comment by jjw24** [comment]
Hi there, I see you added the AppVeyor file in the repo but I don't see any jobs run to publish your release, is it connected?

**Comment by ibnuh** [comment]
> Hi there, I see you added the AppVeyor file in the repo but I don't see any jobs run to publish your release, is it connected?

Hi! I will setup a github actions, pipeline first for the repo to automatically publish a release. Will update here

**Comment by ibnuh** [comment]
I have configured github actions for automated release, please kindly check

**Pull request merged**